modules/shared/src/main/scala/org/tessellation/currency/utils/FeeTransactionUtils.scala

```diff
@@ -0,0 +1,58 @@
+package org.tessellation.currency.utils
+
+import cats.Applicative
+import cats.data.NonEmptyList
+import cats.effect.Async
+import cats.syntax.all._
+
+import org.tessellation.currency.dataApplication.{DataTransaction, DataUpdate, FeeTransaction}
+import org.tessellation.security.signature.Signed
+
+import derevo.circe.magnolia.{decoder, encoder}
+import derevo.derive
+import io.circe.{Decoder, Encoder}
+import org.http4s._
+import org.http4s.circe.jsonOf
+
+object FeeTransactionUtils {
+  def decoderFeeTransactionRequest[F[_]: Async, D <: DataUpdate](
```

Comment:
as above. decoder and encoder should be in the companion object or a separate Ops trait. "Utils" is a bad practice :)

Reply:
👍🏼